src/metabase/query_processor/middleware/process_userland_query.clj

```diff
@@ -34,6 +35,17 @@
 ;;; |                                              Save Query Execution                                              |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(defonce ^:private
+  field-usages-queue
+  (delay (grouper/start!
+          (fn [field-usages]
+            (try
+              (t2/insert! :model/FieldUsage field-usages)
+              (catch Throwable e
+                (log/error e "Error saving field usages"))))
+          :capacity 500
+          :interval (* 20 1000))))
```

Comment:
Is there a reason you're choosing not to extract constants for the intervals, e.g. `fied-usage-interval-seconds` at the top of the namespace? I've nagged a few times, so maybe you disagree with this?

Reply:
Now that I think of it, I wonder whether we shouldn't just make these internal settings.

Reply:
>  just make these internal settings

I think I agree with this, but every now and then I'm reminded of https://www.notion.so/metabase/Design-principles-of-the-product-8c72f4619f5c4e4cb7510e0f66015a08?pvs=4#7f1ad187ccb744308fc86d49c43a4f1d

Reply:
> I've nagged a few times, so maybe you disagree with this?

not at all, I think I had this stashed but removed it by accident. updated now.